src/shared/components/search/SearchContainer.tsx

```diff
@@ -104,6 +104,14 @@ export const cannotQueryMessages = {
     'Filtering Align results is not possible as all of its sequences constitute the alignment.',
 };
 
+const reSearchSpecialWords = /(^| )(or|and|not) /gi;
+const rawQueryClean = (raw: string) =>
+  raw
+    // "     app " -> "app"
+    .trim()
+    // "not app and human oR whatever" -> "NOT app AND human OR whatever"
+    .replaceAll(reSearchSpecialWords, (match) => match.toUpperCase()) || '*';
```

Comment:
```suggestion
    .replace(reSearchSpecialWords, (match) => match.toUpperCase()) || '*';
```
Thinking that as it's a global search `replace` will be sufficient